.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: c
-script: asdf plugin test nodejs . --asdf-plugin-gitref $TRAVIS_BRANCH node --version
+script: asdf plugin test nodejs . --asdf-plugin-gitref HEAD node --version
```

Comment:
I believe the `$TRAVIS_BRANCH` is correct. `HEAD` doesn't appear to be the right commit hash for the latest commit on your branch.

Reply:
`$TRAVIS_BRANCH` points to the base branch name (`master`) instead of my branch name, see:

- https://unhashable.com/getting-the-current-branch-name-during-a-pull-request-in-travis-ci/
- https://github.com/travis-ci/travis-ci/issues/6652

The PR failed before, because `asdf plugin test` checks out the base branch name before running the tests.

I would probably work if I made the pull request from my master branch as well :-) But that should not be necessary

Reply:
@Stratus3D You can see the issue in this build:

https://travis-ci.org/github/asdf-vm/asdf-nodejs/jobs/686053611

`HEAD` is a merge commit between my pr branch and the base branch:

<img width="925" alt="Screenshot 2020-05-12 at 12 42 15" src="https://user-images.githubusercontent.com/235886/81674925-42f60200-944e-11ea-8c62-4c23ecb8e38d.png">

But `$TRAVIS_BRANCH` evaluates to `master`, which is checked out, leaving my changes behind.

Reply:
That makes sense. I understand now. The code you had was correct. Thanks for figuring this out!

Reply:
You're welcome! Thank you for being so pleasant :-)

Was nice to get to the bottom of it.